test/test_integration.py

```diff
@@ -189,6 +193,8 @@ def test_vault_ssl():
         cl.write("secret/ssl/certificate", value=cert)
         cl.write("secret/ssl/key", value=key)
         cl.write("secret/db/password", value="s3cret")
+        cl.write("secret/github/id", value="ghid")
+        cl.write("secret/github/secret", value="ghs3cret")
```

Comment:
Can we check that these actually make it through into the app configuration?  Something like https://github.com/vimc/orderly-web-deploy/blob/fcd1f6ab26572e100b760bbd3663fda84344c8fb/test/test_integration.py#L62-L64